scripts/lxcfs-remount.sh

```
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
-set -exo pipefail
+set -x
 if [[ -z "$1" ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)